pkg/sdn/plugin/node.go

```diff
@@ -60,7 +61,11 @@ type OsdnNode struct {
 	kubeletInitReady   chan struct{}
 	iptablesSyncPeriod time.Duration
 	mtu                uint32
+
+	// Synchronizes operations on egressPolicies
+	egressPoliciesLock sync.Mutex
```

Comment:
(it would be nice to see this as a separate commit since it's fixing a separate pre-existing bug)